charts/kong/values.yaml

```
@@ -182,6 +182,9 @@ admin:
     # - secretName - the secret must contain a key named "tls.crt" with the PEM-encoded certificate.
     # - caBundle (PEM-encoded certificate string).
     # If both are set, caBundle takes precedence.
+    # If neither are set, and the controller uses generated client certificates
+    # ($.Values.ingressController.adminApi.tls.client.enabled=true and $.Values.ingressController.adminApi.tls.client.certProvided=false),
```

I think it's fine to skip the `$.Values.` in the documentation for conciseness.
```suggestion
    # (ingressController.adminApi.tls.client.enabled=true and ingressController.adminApi.tls.client.certProvided=false),
```